web/src/pages/profile.tsx

```diff
@@ -516,6 +520,11 @@ const ProfilePage = (props: Props): ReactElement => {
         </ProfileField>
 
         <ProfileField fieldName="municipality" locked={shouldLockMunicipality()}>
+          {profileData.industryId === "cannabis" && (
```

Comment:
[dust] the thought occurs that maybe this would be a cool concept to make super flexible and reusable at some point with regards to letting content choose to have industry specific alerts in profile when and where ever they want somehow. 

Not relevant at the moment but if they request something like this done another time something to consider maybe

Reply:
That could be a cool feature and it wouldn't be too hard.